src/components/notification-list-item.component.js

```diff
@@ -59,9 +67,9 @@ export class NotificationListItem extends Component {
 
   getIconName = type => {
     switch (type) {
-      case 'commit':
+      case 'Commit':
```

Comment:
Is this a case bug that you find out?

Reply:
yes, the type was wrong.

Reply:
optional: I think it will be good to transform `type` to lowercase and to the check, this way we cover ourselves from any future change.
What do you think?

Reply:
Agree, but should be done in another PR.